README.md

```diff
@@ -567,6 +572,14 @@ method has a `name` property set to one of the following:
 |   `SUBSCRIBE_TO_SESSION_ERROR` | The test encountered an unknown error while attempting to subscribe to a test stream. | 
 |   `SUBSCRIBER_GET_STATS_ERROR` | The test failed to get audio and video statistics for the test stream. | 
 
+#### Errors thrown by the OTNetworkTest.checkCameraSupport() method
```

Comment:
Thanks for adding this method.

Please add a section describing this new `OTNetworkTest.checkCameraSupport()` method under the "OTNetworkTest.testConnectivity()" section (under "API reference" in the readme).

Can you add some code to the sample app that disables the `fullHdCheckbox` element if FHD camera support is not available?

Reply:
@jeffswartz Let's push this in another PR if we need this feature